src/Footer.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const Footer = () => {
+  const footerDetails = [
```

Comment:
For this step of the homework

```
Create a <Footer /> component which should be rendered at the bottom of the page. **Pass the following array as a prop to this component**: ["123 Fake Street, London, E1 4UD", "hello@fakehotel.com", "0123 456789"]. Inside the component, use the data you passed as a prop to render a <ul> list with each item of the array displayed as a <li>.
```

The important bit is that this data should be passed as a **Prop** to this component similar to how `FakeBookings` is passed to `SearchResults` in the `Bookings.js` file.

Could you fix this?

Reply:
Hi Chris, thank you for taking your time to go through my codes and highlighting areas for improvement. I have now made the changes you recommended